### DISCUSSION TITLE
ESP32 type?

### DISCUSSION THREAD

**Discussion by ClearlyDazed** [tronikos/esphome-magnetometer-water-gas-meter#16] [Q&A]
What is the minimum requirements for the ESP32 for this project. There are so many types. Same question for the ESP8266.

**Comment by tronikos** [comment]
Any microcontroller compatible with ESPHome will work https://esphome.io/index.html#supported-microcontrollers
Any ESP8266 will work fine. Unless you want to monitor a high flow meter at 30GPM in which case any ESP32 will give you more accurate results.
